src/main/webapp/scripts/ajax-build-flow.js

```diff
@@ -0,0 +1,12 @@
+setInterval(loadBuildFlow, buildFlowRefreshInterval);
```

Comment:
I suggest that we guard this with:
```
if (Number.isInteger(buildFlowRefreshInterval) && buildFlowRefreshInterval > 0) {
    setInterval(loadBuildFlow, buildFlowRefreshInterval);
}
```
E.g.: test writing setInterval(alert, undefined) in your developer console (please do it in an unimportant window and not in the review window... just saying, you might lose some review comments...)